### PR TITLE
Dashboard default credentials substitution in kong.yml

### PR DESCRIPTION
**Pull request by Lan-Hekary**
## What kind of change does this PR introduce?

This is an update for PR #15980 after adding Basic Auth to the Dashboard (through Kong) in PR #16097
Now the default Dashboard username and password is set in the `.env` file and ignored by git along side the `anon` and `service_role` keys.
I also updated the documentation to reflect the changes accordingly without removing the section about the changing the original `kong.yml` to accommodate multiple users.

### CODE REVIEW

File: apps/docs/pages/guides/self-hosting/docker.mdx [hunk 1]
```diff
@@ -112,7 +112,12 @@ You will need to [restart](#restarting-all-services) the services for the change
 
 ### Dashboard Authentication
 
-The dashboard is protected with Basic Authentication. The default user and password MUST be updated before using Supabase in production. You can update the `./docker/volumes/api/kong.yml` file with your own credentials. For example:
+The dashboard is protected with Basic Authentication. The default user and password MUST be updated before using Supabase in production.
+Update the following values in the `.env` file:
+ - `DASHBOARD_USERNAME`: The default username for the Dashboard 
+ - `DASHBOARD_PASSWORD`: The default password for the Dashboard 
+
+You can Also update the `./docker/volumes/api/kong.yml` file with your own credentials. For example:
```
Comment (soedirgo) [inline apps/docs/pages/guides/self-hosting/docker.mdx:120]: I think we don't need this part

**Review by soedirgo — APPROVED** [review]
Thanks! 🙏

**Comment by Lan-Hekary** [comment]
Hello @soedirgo,
Thanks :D 
Any update on when this will be merged ? also the other one #16681 requires Docs update approval as well.

File: apps/docs/pages/guides/self-hosting/docker.mdx (hunk 1 above)
Comment (soedirgo) [inline apps/docs/pages/guides/self-hosting/docker.mdx:120]:
```suggestion
You can also add more credentials in `./docker/volumes/api/kong.yml`. For example:
```

**Review by kiwicopple — APPROVED** [review]

**Comment by kiwicopple** [comment]
this is great, thanks @Lan-Hekary

**Pull request merged**